site/sfguides/src/data_teams_with_dbt_core/data_teams_with_dbt_core.md

```diff
@@ -0,0 +1,1447 @@
+<<<<<<< HEAD:site/sfguides/src/data_teams_with_dbt_core/data_teams_with_dbt_core.md
```

Comment:
Thank you for the PR! 

This file is broken and does not build. These merge conflicts need to be resolved before merging.